### PR TITLE
fix: target mode visible when disabled

### PR DESCRIPTION
**Pull request by vtsaplin**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,13 +8,14 @@
     "preview": "REACT_APP_VERSION=$npm_package_version aio app run",
     "grammar": "nearleyc web-src/src/helpers/expressions.ne -o web-src/src/helpers/Parser.generated.js",
     "prompts": "node scripts/prompt-generator.js",
-    "examples": "node scripts/zip-examples.js",
+    "prebuild": "npm run prompts && npm run grammar",
```
Comment (basecode) [inline package.json:11]: where is `prebuild` used?

File: package.json [hunk 2]
```diff
@@ -8,13 +8,14 @@
...
     "build": "REACT_APP_VERSION=$npm_package_version aio app build",
-    "deploy:app": "npm run grammar && REACT_APP_VERSION=$npm_package_version aio app deploy --no-actions",
-    "deploy:actions": "aio app deploy --no-web-assets",
-    "deploy": "npm run grammar && npm run prompts && REACT_APP_VERSION=$npm_package_version aio app deploy",
+    "deploy:app": "npm run build && REACT_APP_VERSION=$npm_package_version aio app deploy --no-actions --no-build",
+    "deploy:actions": "npm run build && aio app deploy --no-web-assets --no-build",
+    "deploy": "REACT_APP_VERSION=$npm_package_version aio app deploy",
     "lint": "eslint --quiet 'web-src/src/**/*.js' 'actions/*.js'",
     "fix": "eslint --fix --quiet 'web-src/src/**/*.js' 'actions/*.js'",
+    "pretest": "npm run prompts && npm run grammar",
```
Comment (basecode) [inline package.json:18]: where is `pretest` used?

File: package.json (hunk 1 above)
Comment (vtsaplin) [reply package.json:11]: prebuild is used by build

File: package.json (hunk 2 above)
Comment (vtsaplin) [reply package.json:18]: pretest is used by test

**Review by basecode — APPROVED** [review]

**Pull request merged**